Services/UI/OJS.Services.Ui.Business/Implementations/UsersBusinessService.cs

```diff
@@ -58,5 +59,21 @@ public async Task<bool> IsLoggedInUserAdmin(ClaimsPrincipal userPrincipal)
 
         public async Task AddOrUpdateUser(UserProfile user)
             => await this.usersProfileData.AddOrUpdate<UserProfileServiceModel>(user);
+
+        public async Task<UserAuthInfoServiceModel?> GetAuthInfo()
+        {
+            var currentUser = this.userProvider.GetCurrentUser();
+
+            if (currentUser.IsNull())
```

Comment:
IMO is better to use `== null`, as this will be evaluated by static analysis, whereas `IsNull` does not tell to the compiler that null is handled.